tests/integration/test_chat.py

```diff
@@ -251,6 +251,12 @@
         - Please repeat: step2
 """
 
+yaml_notice = """
+- 1:
+    notice-type: {}
```

Comment:
Nit: Since `{}` is valid yaml, I got confused for a while why you'd want to set the notice type to an empty dict :) Named placeholder (`{notice_type}`) would make it more obvious that it's a placeholder.